src/app/how-council-works/page.tsx

```diff
@@ -41,31 +49,60 @@ export default function HowCouncilWorks() {
+    },
     item: {
       trigger: 'Item',
       content:
-        'An item is any topic or issue listed on a City Council or committee agenda for discussion or decision.',
+        'An item is any topic or issue listed on a City Council or committee agenda for discussion or decision. Items can include a wide range of matters, such as proposed bylaws, policy decisions, or requests for studies.',
     },
     reports: {
       trigger: 'Reports',
       content:
-        'Reports are detailed documents prepared by city staff that provide background information, analysis, and recommendations on specific items.',
+        'Reports are detailed documents prepared by city staff to provide background information, analysis, and recommendations on specific issues.',
     },
     committees: {
       trigger: 'Committees',
       content:
-        'Committees are smaller groups of council members who review and refine items before they go to the full Council for voting.',
+        'A Committee is a smaller group of council members that focus on specific areas like the environment, transportation, or housing. Committees review recommendations from city staff, ask questions, and suggest changes.',
     },
     deputations: {
       trigger: 'Deputations',
       content:
-        'A deputation is a formal presentation where residents can share their views directly with council members during a public meeting.',
+        'Deputations are opportunities for citizens to speak directly to a committee about a specific issue or recommendation being discussed. Deputations allow citizens to share their opinions and influence the decisions being made.',
+    },
+  };
+
+  const imageData = {
+    toronto: {
+      src: '/toronto.png',
+      alt: 'Illustration of Nathan Phillips Square and Toronto City Hall: a scenic depiction of Toronto’s iconic city hall, with the "Toronto" sign in the foreground, people walking and biking, and a fountain in the reflecting pool.',
+    },
+    staff: {
+      src: '/staff.png',
+      alt: 'Illustration of a staff meeting on affordable housing: a group of city staff sits around a table in an office discussing housing policy. Thought bubbles above them depict housing types, consultations, and paperwork.',
+    },
+    committee: {
+      src: '/committee.png',
+      alt: 'Illustration of a committee meeting on affordable housing: a speaker at a podium addresses a panel of business casual committee members, while diverse citizens in the foreground share their opinions via speech bubbles.',
+    },
+    council: {
+      src: '/council.png',
+      alt: 'Illustration of a City Council meeting on affordable housing: the councillors have voted on an item, with the result displayed on the screen showing it has been approved. Touchscreens in front of the councillors indicate votes with green for Approve and red for Reject.',
     },
```

Comment:
Might actually be better to blank out the alt's here since I think these images fit the criteria for decorative/ambiance images
https://www.w3.org/WAI/tutorials/images/decorative/